### PR TITLE
Flatten observations

### PR DESCRIPTION
**Pull request by ptcrowley**
- obs are now flattened
- Base policies now use inherited self.id and self.team from BaseAgentPolicy
- env-light/ added to .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ __pycache__/
 *.pyc
 *.json
 output/
+env-light/
 ctf_gym.egg-info/
```
Comment (makaimann) [inline .gitignore:6]: Good catch on adding `env-light` here. Could you add `env-full` as well?

**Review by makaimann — APPROVED** [review]
Looks good to me -- thanks for the PR! Did you happen to test this with the training code? I'd like to check that before merging.

File: .gitignore (hunk 1 above)
Comment (ptcrowley) [reply .gitignore:6]: Will do! And I did not test this with the training code. I can do that later today and let you know if it works. I am assuming you're referencing the `train_against_easy.py` script?

Comment (makaimann) [reply .gitignore:7]: Yeah testing with that script would be great! I still haven't merged that PR. I'll mark you as a reviewer for it -- hope you don't mind.

Comment (ptcrowley) [reply .gitignore:8]: Just successfully tested the training script. Had to add a few changes to `base_policies.py` (no longer need to index the first element of `obs_batch`), and updated `.gitignore` to include `ray_test` output from trainings.

**Pull request merged**